### ISSUE TITLE
Improve before and after steps in publishing docs

### ISSUE THREAD

**Issue by hannalaakso** [alphagov/govuk-frontend#1957]
# What
We currently have a fairly well documented process for [creating the actual release of GOV.UK Frontend](https://github.com/alphagov/govuk-frontend/blob/master/docs/releasing/publishing.md). 

However it feels like that there are things around it which are missing from the docs:
- We should probably have templates for all the cards we need to create
- The current docs don't mention creating the release notes and getting them 2id for both GOV.UK Frontend or Prototype Kit (forgetting about the latter nearly tripped me up the other day since the release note for the kit hadn't been drafted when I started doing the frontend release - this was my bad but it would have been helpful to have a heads up right at the start of the publishing docs to say this should be done before the Frontend release can be started)
- The current docs don't mention publishing a new version of the kit; again we could check before we start the frontend release whether there's anything pending a merge in the kit that we should try to include in the new release of the kit
- As part of the release we often have guidance/docs in the Design System or Frontend docs that need to be lined up and ready to publish once the release goes out; again it would be good to be reminded about this before starting the release so that we can check whether all these docs have been signed off and are ready to go
- We should probably write the Slack announcement together with the release notes, instead of writing it when the release has already gone out

## Why
To make the release process as easy as possible for us, and to ensure that everything is up-to-date when we tell our users about the new release.

## Who needs to know about this
Mark, developers

**Comment by kellylee-gds** [comment]
@hannalaakso this card is currently in the stretch goals section of the sprint board? Can you add the "done when" section to it please? Thank you!

**Comment by m-green** [comment]
@hannalaakso Just chatted through this with Eoin. When you have a PR ready with the changes, let us know and Eoin and I will review together. If you have any questions in the meantime just let us know!

**Comment by hannalaakso** [comment]
A draft doc here: https://docs.google.com/document/d/1z-atcLlK9e68XKMyv4EVwRKiBzv-oQsD2BtOgfTSOtI/edit

**Comment by EoinShaughnessy** [comment]
@hannalaakso Have reviewed! Let me know if you want to go over anything.

**Comment by EoinShaughnessy** [comment]
Have sent doc for 2i.

**Comment by EoinShaughnessy** [comment]
The content has received 2i approval. I'll raise a PR to update the page.
